tests/unit/serve/executors/test_executor.py

```diff
@@ -88,17 +88,14 @@ def test_executor_with_pymodule_path():
         '''
         )
 
-    import sys
-
-    sys.path.append(cur_dir)
     ex = Executor.load_config(
         '''
     jtype: TestExecutor
     with:
         bar: 123
     metas:
         py_modules:
-            - metas_executors
+            - unit.serve.executors.metas_executors
```

Comment:
can we dedicate one file to this, I would not like to add inter tests dependencies

Reply:
done!